src/plugins/intel_cpu/tests/functional/single_layer_tests/classes/eltwise.cpp

```diff
@@ -0,0 +1,506 @@
+        function = makeNgraphFunction(netType, parameters, eltwise, "Eltwise");
+}
+
+
+TEST_P(EltwiseLayerCPUTest, CompareWithRefs) {
+    run();
+    CheckPluginRelatedResults(compiledModel, std::set<std::string>{"Eltwise", "Subgraph"});
+}
+
+namespace Eltwise {
+
+const ov::AnyMap& additional_config() {
+        static const ov::AnyMap additional_config;
+        return additional_config;
+}
+
+const std::vector<ElementType>& netType() {
+        static const std::vector<ElementType> netType = {
+                ElementType::f32};
+        return netType;
+}
+
+const std::vector<CommonTestUtils::OpType>& opTypes() {
+        static const std::vector<CommonTestUtils::OpType> opTypes = {
+                CommonTestUtils::OpType::VECTOR,
+        };
+        return opTypes;
+}
+
+const std::vector<ngraph::helpers::EltwiseTypes>& eltwiseOpTypesBinInp() {
+        static const std::vector<ngraph::helpers::EltwiseTypes> eltwiseOpTypesBinInp = {
+                ngraph::helpers::EltwiseTypes::ADD,
+                ngraph::helpers::EltwiseTypes::MULTIPLY,
+        #if defined(OPENVINO_ARCH_X86) || defined(OPENVINO_ARCH_X86_64)
+                ngraph::helpers::EltwiseTypes::SUBTRACT,                // TODO: Fix CVS-105430
+                ngraph::helpers::EltwiseTypes::DIVIDE,                  // TODO: Fix CVS-105430
+                ngraph::helpers::EltwiseTypes::FLOOR_MOD,               // TODO: Fix CVS-111875
+        #endif
+                ngraph::helpers::EltwiseTypes::SQUARED_DIFF,
+        };
+        return eltwiseOpTypesBinInp;
+}
+
+const std::vector<ngraph::helpers::EltwiseTypes>& eltwiseOpTypesDiffInp() {
+        static const std::vector<ngraph::helpers::EltwiseTypes> eltwiseOpTypesDiffInp = { // Different number of input nodes depending on optimizations
+        #if defined(OPENVINO_ARCH_X86) || defined(OPENVINO_ARCH_X86_64)
+                ngraph::helpers::EltwiseTypes::POWER,                   //TODO: Fix CVS-111880
+        #endif
+                // ngraph::helpers::EltwiseTypes::MOD // Does not execute because of transformations
+        };
+        return eltwiseOpTypesDiffInp;
+}
+
+const std::vector<ngraph::helpers::EltwiseTypes>& eltwiseOpTypesBinDyn() {
+        static const std::vector<ngraph::helpers::EltwiseTypes> eltwiseOpTypesBinDyn = {
+                ngraph::helpers::EltwiseTypes::ADD,
+                ngraph::helpers::EltwiseTypes::MULTIPLY,
+        #if defined(OPENVINO_ARCH_X86) || defined(OPENVINO_ARCH_X86_64) // TODO: Fix CVS-105430
+                ngraph::helpers::EltwiseTypes::SUBTRACT,
+        #endif
+                ngraph::helpers::EltwiseTypes::SQUARED_DIFF,
+        };
+        return eltwiseOpTypesBinDyn;
+}
+
+const std::vector<CPUSpecificParams>& cpuParams_4D() {
+        static const std::vector<CPUSpecificParams> cpuParams_4D = {
+                //CPUSpecificParams({nChw16c, nChw16c}, {nChw16c}, {}, {}),
```

Comment:
Please remove
Please double check all the extra comments made

Reply:
Fixed